lib/mix/test/mix/tasks/xref_test.exs

```diff
@@ -1092,6 +1135,19 @@ defmodule Mix.Tasks.XrefTest do
       """)
     end
 
+    test "group with multiple unconnected files (json)" do
```

Comment:
IMO we don't need this test. The group works unrespective of the format.

Reply:
ack.